Math/TriangleQuest2.py

```diff
@@ -7,4 +7,4 @@
 Problem   : https://www.hackerrank.com/challenges/triangle-quest-2/problem
 '''
 for i in range(1,int(raw_input())+1): #More than 2 lines will result in 0 score. Do not leave a blank line also
-    print(((10**i)/9)**2)
+    print(round((10**i)/9)**2)
```

Comment:
The challenge is being failed with this line of code using `Python 2`.

![image](https://user-images.githubusercontent.com/5465070/113305697-1cbaa880-9325-11eb-9be6-22e5323770c9.png)